### ISSUE TITLE
Would be nice if field names could be editable from the UI

### ISSUE THREAD

**Issue by lekkas** [resin-io/pensieve#135]

**Comment by LucianBuzzo** [comment]
Agreed, at the moment changing field titles would require 'manual' intervention, by pulling the repo onto your computer, doing a search and replace and committing the changed file back to GitHub. Being able to edit the field using the 'edit schema' section would be a good improvement.

**Comment by alexandrosm** [comment]
@lekkas is this a blocker for using pensieve in agendas?
